frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -547,6 +549,9 @@
             onSuccess: function(response) {
                 var details = response.data;
                 vm.phaseSplits = details;
+                if(details.length == 0) {
+                    vm.privateLeaderboardUnique = true; 
```

Comment:
Can you please change the name of the variable to `vm.isChallengeLeaderboardPrivate = true`